packages/app/lib/apps.dart

```diff
@@ -3,14 +3,10 @@ import 'package:neon_files/neon_files.dart';
 import 'package:neon_news/neon_news.dart';
 import 'package:neon_notes/neon_notes.dart';
 import 'package:neon_notifications/neon_notifications.dart';
-import 'package:shared_preferences/shared_preferences.dart';
 
-List<AppImplementation> getAppImplementations(
-  final SharedPreferences sharedPreferences,
-) =>
-    [
-      FilesApp(sharedPreferences),
-      NewsApp(sharedPreferences),
-      NotesApp(sharedPreferences),
-      NotificationsApp(sharedPreferences),
-    ];
+final List<AppImplementation> appImplementations = [
```

Comment:
Since there is no logic left I would just inline this

Reply:
This isn't possible as we still need it for the screenshot tests